api/drivers/aws/driver.js

```diff
@@ -0,0 +1,266 @@
+/**
+ * Nanocloud turns any traditional software into a cloud solution, without
+ * changing or redeveloping existing source code.
+ *
+ * Copyright (C) 2016 Nanocloud Software
+ *
+ * This file is part of Nanocloud.
+ *
+ * Nanocloud is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Nanocloud is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * AWSDriver
+ *
+ * @description :: Driver for Amazon Web Service EC2 Iaas
+ */
+
+/* global Machine, ConfigService */
+
+const pkgcloud = require('pkgcloud');
+const Promise = require('bluebird');
+const Driver = require('../driver');
+const ursa = require('ursa-purejs');
+const fs = Promise.promisifyAll(require('fs'));
+
+/**
+ * The port on which plaza will be listenning in the instances.
+ */
+const PLAZA_PORT = 9090;
```

Comment:
Plaza port is stored in environment variable for user storage. Shouldn't be the same here ?